app/classifier.py

```diff
@@ -1,18 +1,56 @@
+from operator import itemgetter
+import logging
+import string
+import time
+import os
+
+import numpy as np
+import tensorflow as tf
+
 from app import embedder
+from app import model_fetcher
+
+
+def fetchVocab(vocab: str):
+    with open(vocab, 'r') as f:
+        return f.readlines()
 
 
 class Classifier:
 
-    def __init__(self):
-        self.embedder = embedder.Embedder()
+    def __init__(self, bert, classifier, vocab: str, prefetch=False):
+        self.logger = logging.getLogger('app.logger')
+        if prefetch:
```

Comment:
uhh, this is a slightly annoying API: it makes it the caller's responsibility to worry about local and cloud models to be in sync... Can you do some basic timstamp comparison and remove the prefetch arg?

Reply:
I've removed the prefetch arg.